stock_putaway_method/models/product_strategy.py

```diff
@@ -0,0 +1,13 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class PutAwayStrategy(models.Model):
+    _inherit = 'product.putaway'
+
+    method = fields.Selection('_get_putaway_options', "Method",
```

Comment:
YAPF makes weird decisions sometimes ;)
try to keep formatting with parameters on separate lines with trailing commas )